nss-evening-cohort-14/renaissance-fair-dashboard-renaissance-fair
# Ls event view

## Description
- when the user clicks the event card's "view details" button, it will bring them to a single event
- when a user adds an event, the new event will be added to the events view directly after submitting the form

## Related Issue
#139 

## Motivation and Context
This shows users a single event view for the event. 

## How Can This Be Tested?
- user can click "view details" and see a new view
- user can submit an event and see their new event on the events page directly after

## Types of changes
